saga/order-service/src/main/resources/application.properties

```diff
@@ -0,0 +1,27 @@
+quarkus.datasource.db-kind=postgresql
+quarkus.datasource.jdbc.url=jdbc:postgresql://localhost:5432/orderdb
+quarkus.datasource.username=orderuser
+quarkus.datasource.password=orderpw
+quarkus.hibernate-orm.log.sql=true
+
+quarkus.hibernate-orm.database.generation=drop-and-create
```

Comment:
this setting (also found in other services) takes care about DB schema gen which is why I commented on need for the .sql init files

Reply:
See my other comment, it doesn't create the schema itself unfortunately.